lib/zendesk_apps_support/manifest.rb

```diff
@@ -154,6 +168,12 @@ def set_locations_and_hosts
         end
     end
 
+    def normalize_array_or_string_to_array_or_bool(val)
+      val = [ val ] if val.is_a?(String)
+      val = false if val.is_a?(Array) && val.empty?
```

Comment:
Could the first line return explicitly if val was a string rather than falling through to be reevaluated here?